gallery/src/components/NxCombobox/NxComboboxPage.tsx

```diff
@@ -0,0 +1,251 @@
+              <NxTable.Cell>
+                Whether to render a short variant of the text input box and correspondingly the dropdown.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>disabled</NxCode></NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>false</NxTable.Cell>
+              <NxTable.Cell>
+                When set, this prop disables the text input input and the dropdown.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>emptyMessage</NxCode></NxTable.Cell>
+              <NxTable.Cell>ReactNode</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>"No Results Found"</NxTable.Cell>
+              <NxTable.Cell>
+                The text to display when the user performs a query for which no results are returned.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>inputProps</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Props for <NxCode>NxTextInput</NxCode> except
+                for <NxCode>type</NxCode>,{' '}
+                <NxCode>value</NxCode>,{' '}
+                <NxCode>isPristine</NxCode>,
+                and <NxCode>readOnly</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                Props to apply to the text input within the <NxCode>NxCombobox</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell>HTML <NxCode>&lt;div&gt;</NxCode> Attributes</NxTable.Cell>
+              <NxTable.Cell>
+                <NxTextLink external href="https://developer.mozilla.org/en/docs/Web/HTML/Element/div">
+                  Div Attributes
+                </NxTextLink>
+              </NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                <NxCode>NxCombobox</NxCode> supports any HTML attribute that's normally supported by{' '}
+                HTML <NxCode>&lt;div&gt;</NxCode>.
+              </NxTable.Cell>
+            </NxTable.Row>
+          </NxTable.Body>
+        </NxTable>
+      </NxTile.Subsection>
+      <NxTile.Subsection>
+        <NxTile.SubsectionHeader>
+          <NxH3>Helpers</NxH3>
+        </NxTile.SubsectionHeader>
+        <NxP>
+          If using <NxCode>NxCombobox</NxCode> for a backend query, the search functionality should be used with a
+          debounce in order to prevent excessive queries to the backend. The standard timing value to use for that
+          debounce is provided via the <NxCode>NX_SEARCH_DROPDOWN_DEBOUNCE_TIME</NxCode> export.
```

Comment:
Now that `NX_SEARCH_DROPDOWN_DEBOUNCE_TIME` is relevant to more components than just `NxSearchDropdown`, should it be renamed?

Reply:
The name here doesn't get me confused since combobox has an editable input and a dropdown. But I could see it would be cleaner if we have a name fit with component.
To clarify your suggestion, do you mean
1) Changing the name from the export in index.ts? something like `NX_STANDARD_DEBOUNCE_TIME` and updated all associated files.
2) Changing the name used in combobox component only?

Reply:
> Changing the name from the export in index.ts? something like NX_STANDARD_DEBOUNCE_TIME and updated all associated files.

Yeah

Reply:
Thanks for confirmation. It is updated here bc7665db4071d11fb355643af7c63a21bab723bb.